luigi/util.py

```diff
@@ -283,21 +283,24 @@ def run(self):
                # ...
     """
 
-    def __init__(self, *tasks_to_inherit):
+    def __init__(self, *tasks_to_inherit, ignore_collisions=[]):
         super(inherits, self).__init__()
         if not tasks_to_inherit:
             raise TypeError("tasks_to_inherit cannot be empty")
 
         self.tasks_to_inherit = tasks_to_inherit
+        self.ignore_collisions = ignore_collisions
 
     def __call__(self, task_that_inherits):
-        # Get all parameter objects from each of the underlying tasks
+
+        # Check for parameter collisions and raise an exception if found.
+        self.check_for_parameter_collisions(task_that_inherits)
+
+        # Get all parameter objects from each of the underlying tasks and
+        # them to the inheriting task.
```

Comment:
maybe a typo... not sure what you are trying to say `and them to the inheriting task`

Reply:
yes typo, thanks. Was supposed to read: `and add them to the inheriting task`. Will make the change.